### PR TITLE
Release mode: updating Changelog and Authors

### PR DESCRIPTION
**Pull request by Garyfallidis**
:dancers:

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -24,6 +24,19 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 0.9.0 (Wednesday, 18 March 2015)
+
+* Anatomically Constrained Tissue Classifiers for Tracking
+* Massive speedup (1.5X to 6X depending on data) of Constrained Spherical Deconvolution (CSD)
```
Comment (arokem) [inline Changelog]: => "(1.5x to 6x, depending on the data)"

Comment (Garyfallidis) [inline Changelog]: lol okay will remove that.

Comment (Garyfallidis) [inline Changelog]: Give me a sec.

Comment (arokem) [inline Changelog]: No need to remove - just add a comma!

File: Changelog [hunk 2]
```diff
@@ -24,6 +24,19 @@ Dipy
...
+
+* Anatomically Constrained Tissue Classifiers for Tracking
+* Massive speedup of Constrained Spherical Deconvolution (CSD)
+* Recursive calibration of response function for CSD
+* New experimental framework for clustering
+* Improvements and 10X speedup for Quickbundles
+* Improvements in Linear Fascicle Evaluation (LiFE)
+* New implementation of Geodesic Anisotropy 
+* New efficient transformation functions for registration
+* Sparse Fascicle Model now supports multiple b-values too
```
Comment (arokem) [inline Changelog]: How about "now supports acquisitions with multiple b-values"?

**Pull request merged**